crawl-ref/source/mon-death.cc

```diff
@@ -1039,6 +1040,22 @@ static void _infestation_create_scarab(monster* mons)
     infestation_death_fineff::schedule(mons->pos(), mons->name(DESC_THE));
 }
 
+static void _pharaoh_ant_bind_souls(monster *mons)
+{
+    simple_monster_message(*mons, " binds the souls of nearby monsters.");
+    for (monster_near_iterator mi(mons, LOS_NO_TRANS); mi; ++mi)
+    {
+        if (*mi == mons)
```

Comment:
Do we need this check? Surely pharaoh ants can't bind themselves regardless, since they're already undead, right?